### ISSUE TITLE
Importing a launchdarkly_project results in incomplete import

### ISSUE THREAD

**Issue by joekarl** [terraform-providers/terraform-provider-launchdarkly#21]
When importing a project (`terraform import launchdarkly_project.example example-project`) the import runs successfully and is added to the terraform state. However, since resource_launchdarkly_project.go is using `ImportStatePassthrough`, only the ID is imported. This means on the next plan/apply the project will be destroyed/recreated because the `key` attribute is `ForceNew` and wasn't imported.

State snipped post import
```
{
  "mode": "managed",
  "type": "launchdarkly_project",
  "name": "ld-project",
  "provider": "provider.launchdarkly",
  "instances": [
   {
       "schema_version": 0,
       "attributes": {
        "environments": [],
        "id": "example-project",
        "key": "",
        "name": "",
        "tags": []
      },
      "private": "xxxxxxxxxxxxxxxxxx"
    }
  ]
}
```

Plan post import
```
 Terraform will perform the following actions:
  # launchdarkly_project.ld-project must be replaced
-/+ resource "launchdarkly_project" "ld-project" {
      ~ id   = "example-project" -> (known after apply)
      + key  = "example-project" # forces replacement
      + name = "Example Project"
      - tags = [] -> null
      + environments {
          + api_key              = (sensitive value)
          + client_side_id       = (sensitive value)
          + color                = (known after apply)
          + confirm_changes      = (known after apply)
          + default_track_events = (known after apply)
          + default_ttl          = (known after apply)
          + key                  = (known after apply)
          + mobile_key           = (sensitive value)
          + name                 = (known after apply)
          + require_comments     = (known after apply)
          + secure_mode          = (known after apply)
          + tags                 = (known after apply)
        }
    }
Plan: 1 to add, 0 to change, 1 to destroy.
```

=========================================================

Expected import
```
{
  "mode": "managed",
  "type": "launchdarkly_project",
  "name": "ld-project",
  "provider": "provider.launchdarkly",
  "instances": [
   {
       "schema_version": 0,
       "attributes": {
        "environments": [],
        "id": "example-project",
        "key": "example-project",
        "name": "",
        "tags": []
      },
      "private": "xxxxxxxxxxxxxxxxxx"
    }
  ]
}
```

Expected plan
```
No changes. Infrastructure is up-to-date.
```

**Comment by jkdey** [comment]
Running into the same issue in 1.2.0, 1.1.0 is fine

**Comment by ldhenry** [comment]
Hi @joekarl and @jkdey,

Thanks for bringing this to my attention. As @jkdey suggests, this bug was introduced in v1.2.0 and v.1.1.0 should work for now. We will work to get a fix out ASAP.

Thanks,
Henry

**Comment by ldhenry** [comment]
Hi @joekarl and @jkdey,

I just released a fix for this (v1.2.1). Feel free to re-open this issue if you are still experiencing trouble.

Thanks,
Henry

**Comment by joekarl** [comment]
Thanks @ldhenry !
